.travis.yml

```diff
@@ -4,8 +4,8 @@ jdk:
 - oraclejdk8
 
 scala:
-- 2.11.11
-- 2.12.2
+- 2.11.8
```

Comment:
I reverted to older versions of Scala so that I could get Typelevel Scala in order to use the literal types flag.